scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -1304,6 +1306,47 @@ private void updateInheritedPropagatedVariables(Map<String, Serializable> propag
         }
     }
 
+    @Override
+    public boolean isWallTimeSet() {
+        return getRuntimeWallTime() > 0;
+    }
+
+    protected long getRuntimeWallTime() {
```

Comment:
rename `getRuntimeWallTime` in `getRuntimeWalltime`

Reply:
I made it match the syntax of the existing method getWallTime